test/windows/application-scans.test.ts

```diff
@@ -0,0 +1,81 @@
+import * as path from "path";
```

Comment:
what's the difference in the windows specific tests? why do we need them?

Reply:
There isn't much difference, just normalising the paths, etc. We've decided to go this way, as it wasn't feasible to run all the tests on windows, lots of them are irrelevant and/or we didn't quite work out why they are failing (like the docker socket one). So the best solution for now was to have few dedicated tests, covering high level paths. So there's duplication but I think it's nicer than having a check if platform is windows in every test, wdyt?